freeglut/freeglut/src/mswin/fg_spaceball_mswin.c

```diff
@@ -1,199 +1,612 @@
-/*
```

Comment:
This whole file used to have UNIX line endings, and your patch changes it to DOS line endings, making it extremely hard to see what the actual changes are. The whole file is treated as completely replaced by git as you can see. Please fix the line endings to make it possible to pinpoint changes, and to review it properly.